directory.py

```diff
@@ -0,0 +1,21 @@
+import json
+import os
+import subprocess
+
+os.system("source venv/bin/activate")
```

Comment:
I could be wrong, this looks like it can/should be removed.  The 'activate' happens, but in a different context- a separate process